client/post-editor/style.scss

```diff
@@ -61,6 +61,32 @@
 	}
 }
 
+.editor__site {
+	display:  flex;
+	background: lighten( $gray, 30% );
+
+	@include breakpoint( ">660px" ) {
+		display: none;
+	}
+
+	.site__title,
+	.site__domain {
+		&::after {
+			@include long-content-fade( $color: lighten( $gray, 30% ) );
+		}
+	}
+
+	.site__home {
```

Comment:
Afaik it's there and visible (not on hover) because on mobile there wouldn't be any way to see if that's something that leads somewhere.

In this context however I feel we can drop it. We already have a preview link, and I think it's more important in this context to see the site icon — sure, it stays tappable if you try as a hidden feature. :)